### DISCUSSION TITLE
Etcd 3.5.13 failed to start with error message 'cannot fetch cluster info from peer URLs: could not retrieve cluster information from the given URLs'

### DISCUSSION THREAD

**Discussion by carller0302** [etcd-io/etcd#18892] [General]
![1731579652663](https://github.com/user-attachments/assets/b401b5d6-c645-452e-b21a-41bb6e94b213)
![1731579818391](https://github.com/user-attachments/assets/ba682fdf-e063-4651-ba60-e0c402b1464d)

**Comment by ahrtr** [comment]
You did not correctly configure the parameters when starting etcd cluster.

- If you start a brand new etcd cluster,  the value for `--initial-cluster-state` should be `new` for each member;
- If you add a new member into an existing cluster, the value for `--initial-cluster-state` should be `existing` , and you also need to configure correct `--initial-cluster` when starting the new member.
